app/models/assignment.rb

```diff
@@ -0,0 +1,8 @@
+class Assignment < ApplicationRecord
+  # User assigned to do the describing
+  belongs_to :assignee, class_name: "User"
+  # User assigned to be desribed
+  belongs_to :assigned, class_name: "User"
+
+  enum status: %w(open closed)
+end
```

Comment:
re: open/closed I imagine this could be a way to find users who did not complete the exercise yet.